.github/workflows/kickstart-tests.yml

```diff
@@ -55,10 +55,12 @@ jobs:
           REF_DEVEL='f'$FEDVERS'-devel'
           REF_RELEASE='f'$FEDVERS'-release'
 
-          if [ "$TARGET_BRANCH" == "$REF_DEVEL" ] || [ "$TARGET_BRANCH" == "$REF_RELEASE" ] ; then
-            echo "::set-output name=lorax_build_container::fedora:$FEDVERS" ;
-          elif [ "$TARGET_BRANCH" == "master" ] ; then
-            echo "::set-output name=lorax_build_container::fedora:rawhide" ;
+          if [ "$TARGET_BRANCH" == "$REF_DEVEL" ] || [ "$TARGET_BRANCH" == "$REF_RELEASE" ]; then
+            echo "::set-output name=lorax_build_container::fedora:$FEDVERS";
+          elif [ "$TARGET_BRANCH" == "master" ]; then
+            echo "::set-output name=lorax_build_container::fedora:rawhide";
```

Comment:
Not critical, but - if you want to remove the spaces, maybe do so in another commit? The diff is very hard to understand because all the lines change, but there is no change to be seen in the part I highlight here. (Arguably that's a Github problem, Gitlab could display than nicely.)